tests/examples/pacer/nef/s3/mdb_2.json

```diff
@@ -16,7 +16,7 @@
           "pacer_doc_id": "092051714783",
           "pacer_magic_num": "11582823",
           "pacer_seq_no": "409",
-          "short_description": "Order on Motion To Quash"
+          "short_description": "Order on Motion To Quash - CH"
```

Comment:
This one shouldn't have the lagging " - CH"
![image](https://github.com/user-attachments/assets/19274fa4-bba9-4b16-88b5-2370b23a364a)